src/Sql/dbo/Stored Procedures/OrganizationUser_ReadExistingByOrganizationIdEmail.sql

```diff
@@ -0,0 +1,30 @@
+CREATE PROCEDURE [dbo].[OrganizationUser_ReadExistingByOrganizationIdEmail]
```

Comment:
Same here, `Existing` in my opinion doesn't provide any additional context and could be left out.

Reply:
Similar response to above, but also just `ReadByOrganizationId` would be expected to return the full OrganizationUser object. My name is no better on this front. I think I like the Filter or Select options above.